src/App.js

```diff
@@ -14,11 +15,79 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedProduct: [],
+  }
+
+  addProduct = (event, product) => {
+    const element = event.target.closest('li');
+
+    element.classList.add('is-success');
+    this.setState(state => ({
+      selectedProduct: [...state.selectedProduct, product],
+    }));
+  }
+
+  removeProduct = (event, product) => {
+    const element = event.target.closest('li');
+
+    element.classList.remove('is-success');
+
+    if (element.childNodes[0].textContent === product) {
+      this.setState((prevState) => {
+        const index = prevState
+          .selectedProduct.findIndex(item => item === product);
+
+        if (index !== -1) {
+          prevState.selectedProduct.splice(index, 1);
+        }
+
+        return {
+          selectedProduct: prevState.selectedProduct,
+        };
+      });
+    }
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <h1 className="title is-2">
+          {`Selected good: - ${this.state.selectedProduct.join(', ')}`}
+        </h1>
+        <ul>
+          {goodsFromServer.map((item, index) => (
+            <>
+              <li className="subtitle is-5 button" key={item}>
```

Comment:
you can add a condition that decides to add a class or not, don't use .classList.add and DOM-elements at all